doc/cla/individual/daramousk.md

```diff
@@ -0,0 +1,11 @@
+Greece, 2017-01-22
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+George Daramouskas george.daramouskas@gmail.com https://github.com/daramousk
```

Comment:
Thank you for the CLA signature but it seems there is still an issue.

- you are signing using the address `g...@targetintegration.com`
- you are signing for the address `g...@gmail.com`

The email address in the CLA must be the same as for the signature for the verification to succeed.
You can either change the your [git configuration](https://help.github.com/articles/setting-your-email-in-git/) or modify the email in the CLA signature.